server/src/main/java/za/dots/db/DatasourceConnection.java

```diff
@@ -0,0 +1,21 @@
+package za.dots.db;
```

Comment:
Could we name the package `dao`? DAO is used for database operations, and isolates it from business and logic.